internal/deviations/deviations.go

```diff
@@ -74,6 +74,11 @@ func P4RTMissingDelete(_ *ondatra.DUTDevice) bool {
 	return *p4rtMissingDelete
 }
 
+// P4RTSkipStreamError returns whether Stream Disconnect for UnsetElectionID needs to be skipped.
```

Comment:
Comment is throwing static analysis error.

Also can we unexport the `P4rtUnsetElectionIDUnsupported` flag. We don't want users to be able to use the method as well as the function.

Reply:
oh my bad, i modified the comment and then made the variable as p4rtUnsetElectionIDUnsupported